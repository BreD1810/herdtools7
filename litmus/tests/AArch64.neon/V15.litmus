AArch64 V15
(* Tests MOVI scalar variant *)
{
0:X0=x;0:X1=y;
}

 P0                     ;
 MOVI D0,#255    ;
 ST1 {V0.B}[0], [X0]    ;
 ST1 {V0.B}[1], [X1]    ;
 forall (x=255 /\ y=0)
