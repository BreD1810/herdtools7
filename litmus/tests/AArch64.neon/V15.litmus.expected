Test V15 Required
Histogram (1 states)
1000000:>x=255; y=0;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (x=255 /\ y=0) is validated
Hash=7428d443bbf832d5f78973f326e0759a
Observation V15 Always 1000000 0
